jobs/pull-kubernetes-e2e-gce-cri.sh

```diff
@@ -50,7 +50,7 @@ export GINKGO_PARALLEL="y"
 # This list should match the list in kubernetes-e2e-gce.
 export GINKGO_TEST_ARGS='--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]'
 export FAIL_ON_GCP_RESOURCE_LEAK="false"
-export PROJECT="kubernetes-pr-cri-validation"
+export PROJECT="${PROJECT:-kubernetes-pr-cri-validation}"
```

Comment:
If you'd prefer I back out the changes to `KUBE_GCS_RELEASE_BUCKET` and `PROJECT`, I can do that, although I think there's a benefit to being able to run `bootstrap.py` locally and have it point to your own project and bucket.